cp_segmentation.xml

```diff
@@ -43,12 +43,12 @@
             </param>
             <when value="nuclei">
                 <param name="chan" value="0" type="integer" optional="true" label="Type in the channel to segment"
-                    help="Integer. If 0, all channels are fused; if greater than 0, corresponds to the channel index number (starting from 1) to be extracted. Leave blank for None"/>
+                    help="If 0, all channels are fused; if greater than 0, corresponds to the channel index number (starting from 1) to be extracted. Leave blank for None"/>
                 <param name="chan2" value="0" type="hidden" />
             </when>
             <when value="cyto">
                 <param name="chan" value="0" type="integer" optional="true" label="Type in the channel to segment" help="0=grayscale, 1=red, 2=green, 3=blue"/>
-                <param name="chan2" value="0" type="integer" optional="true" label="Type in the channel for nuclei segmatation (Optional)" help="0=None (will set to zero), 1=red, 2=green, 3=blue"/>
```

Comment:
@qiagu should those params maybe modelled as select box?

Reply:
You mean selection from 0, 1, 2, 3? That's true for the official cellpose. We have to extend the tool to support multiplex images which have up to 40 or more channels. My plan is the extract the channel, save it to a intermediate image with 1-2 channel, then call for the cellpose commands.

Reply:
I just saw the help, and this indicates a list between 0-3 ...

Reply:
That help is what official documentation says and cellpose seems to be designed to support RGB, simple images only. Multiplex images in our group go through the hidden feature.

Reply:
Actually, the the channel extraction work is not implemented yet. We use a different hacky way to bypass the channel limit in older versions. Now change is needed.